### PR TITLE
Batch ornaments settings on iOS

### PR DESCRIPTION
**Pull request by evil159**
Instead of applying ornament options one by one batch them up and apply all at once(per ornament) to avoid unnecessary updates triggered by its setter.

### CODE REVIEW

**Comment by yunikkk** [comment]
@evil159 fyi conflicts here

File: ios/Classes/CompassController.swift [hunk 1]
```diff
@@ -4,36 +4,33 @@ import UIKit
...
         if let visible = settings.enabled {
             if !visible.boolValue {
-                mapView.ornaments.options.compass.visibility = OrnamentVisibility.hidden
-            } else {
-                if settings.fadeWhenFacingNorth?.boolValue ?? true {
-                    mapView.ornaments.options.compass.visibility = OrnamentVisibility.adaptive
-                } else {
-                    mapView.ornaments.options.compass.visibility = OrnamentVisibility.visible
-                }
+                compass.visibility = visible.boolValue ? .adaptive : .hidden
```
Comment (yunikkk) [inline ios/Classes/CompassController.swift:29]: Hmm is it correct if's sequence?

Comment (evil159) [reply ios/Classes/CompassController.swift:29]: Ah, true, I missed the `fadeWhenFacingNorthPart` 🤦

Comment (evil159) [reply ios/Classes/CompassController.swift]: Should be better now

**Review by yunikkk — APPROVED** [review]
LGTM!

**Comment by yunikkk** [comment]
@evil159 auto rebase does not pass, 
```
Update branch attempt failed
Rebase conflict between head and base.
```
guess needs manual

**Pull request merged**